htdocs/web_portal/views/site/edit_api_auth.php

```diff
@@ -41,23 +53,36 @@
     <form class="inputForm" method="post" action="index.php?Page_Type=Edit_API_Authentication_Entity&parentid=<?php echo($params['site']->getId())?>&authentityid=<?php xecho($params['authEnt']->getId())?>" name="addAPIAuthenticationEntity">
         <div style="margin-bottom: 0.5em;">
             <span class="input_name">Identifier (e.g. Certificate DN or OIDC Subject)*</span>
-            <input type="text" value="<?php xecho($params['authEnt']->getIdentifier()) ?>" name="IDENTIFIER" class="input_input_text">
+            <input
+                type="text"
+                value="<?php xecho($params['authEnt']->getIdentifier()) ?>"
+                name="IDENTIFIER"
+                class="input_input_text"
+                <?php echo $params['isRenewalRequest'] ? 'disabled' : ''; ?>
+            >
         </div>
+
         <div style="margin-bottom: 0.5em;">
             <span class="input_name">Credential type*</span>
-            <select name="TYPE" class="input_input_text">
+            <select
+                name="TYPE"
+                class="input_input_text"
+                <?php echo $params['isRenewalRequest'] ? 'disabled' : ''; ?>
+            >
                 <?php foreach($params['authTypes'] as $authType) {?>
                     <option value="<?php xecho($authType) ?>"<?php if ($params['authEnt']->getType() == $authType) {echo " selected=\"selected\"";} ?>>
                         <?php xecho($authType) ?>
                     </option>
                 <?php } ?>
             </select>
         </div>
+        <?php if (!($params['isRenewalRequest'])) {?>
         <div style="margin-bottom: 1em">
             <div class="input_warning">
-                WARNING: it is possible to delete information using the write functionality of the API. Leave Allow API write unchecked if
-                you do not need to write data.
+                WARNING: It is possible to delete information using the write functionality of the API.
+                Leave Allow API write unchecked if you do not need to write data.
```

Comment:
A slightly long line

```suggestion
                WARNING: It is possible to delete information using the write
                functionality of the API. Leave Allow API write unchecked
                if you do not need to write data.
```

Reply:
As discussed, adhering to limit now. Done